index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <section class="navigation" >
+        <h1 id="name">John Doe</h1>
+        <div id="menu"><a href=""><img src="images/menu.png" alt=""></a></div>
+    </section>
+    <section class="body">
+        <h3 id="one">Hey There.
+            I’m Alex</h3>
```

Comment:
Consider making this heading as h1, since the introduction should be the first thing visible.